libevm/ExtVMFace.cpp

```diff
@@ -84,14 +84,21 @@ size_t getCodeSize(evm_context* _context, evm_address const* _addr)
     return env.codeSizeAt(fromEvmC(*_addr));
 }
 
-size_t getCode(byte const** o_code, evm_context* _context, evm_address const* _addr)
+size_t copyCode(evm_context* _context, evm_address const* _addr, size_t _codeOffset,
```

Comment:
Oh, I think we made a mistake. This limits code size to 64k.

Reply:
No clue what you are talking about.

Reply:
Still no clue.

Reply:
Never mind, had an embedded system in mind where `size_t` is 16 bits.

I think it would make sense setting an expectation (`static_assert`) that `size_t` is at least 32 bits wide.

Reply:
Where is the assert? For ewasm it can certainly be bigger than 65k (think about "system contracts").

Reply:
There is no assert, but size_t is 32-bit at worse if you target 32-bit ARM architectures. In practice `size_t` is the size of CPU registers.